frontend/src/app/testResults/mocks/resultsMultiplex.mock.tsx

```diff
@@ -96,7 +96,7 @@ const TEST_RESULTS_MULTIPLEX = [
     internalId: "7c768a5d-ef90-44cd-8050-b96dd7aaa1d5",
     dateTested: "2021-03-19T19:27:21.052Z",
     result: "POSITIVE",
```

Comment:
We'll need to take these `result` mocks out once the graphQL backend is updated, right?

Reply:
Yes, thank you for the reminder! 😸 I was planning on doing that along with these changes https://github.com/CDCgov/prime-simplereport/pull/4259